### ISSUE TITLE
detritus left over from moving packages to repository directory

### ISSUE THREAD

**Issue by dalehenrich** [SeasideSt/Seaside#855]
I noticed the following issues in the directory structure:
```
repository/properties.json                                   # delete
repository/monticello.meta                                 # delete
.gitattributes                                                       # move to repository dir
.filetree                                                              # delete
repository/*Javascript-Pharo20-Core.package #delete
```

**Comment by marschall** [comment]
That's done, right?

**Comment by marschall** [comment]
Closed by #856
